pkgs/by-name/re/resticprofile/package.nix

```diff
@@ -50,11 +50,8 @@ buildGoModule rec {
     rm priority/ioprio_test.go # tries to set nice(2) IO priority
     rm restic/downloader_test.go # tries to use network
     rm schedule/schedule_test.go # tries to use systemctl
-
-    # `config/path_test.go` expects `$HOME` to be the same as `~nixbld` which is `$NIX_BUILD_TOP`
-    export HOME="$NIX_BUILD_TOP"
-    # `util/tempdir_test.go` expects `$HOME/.cache` to exist
-    mkdir "$HOME/.cache"
+    rm config/path_test.go # expects normal environment
```

Comment:
Shouldn't those then be conditional on darwin?

Reply:
Disabling these tests removes unnecessary logic, that, IMO, just clutters the expression and is very hacky.

These tests just mostly just check that the user running them uses a normal file system with regular permissions.

I don't think we lose much by removing these tests.